ci/build_wheel.sh

```diff
@@ -0,0 +1,28 @@
+#!/bin/bash
+# Copyright (c) 2023, NVIDIA CORPORATION
+
+# install CUDAToolKit version 12.2
+apt-get update
+apt-get -y install cuda-toolkit-12-2
+
+rapids-logger "Check conda environment"
+
+conda list
+
+rapids-logger "Building wheel"
+
+pip wheel .
+
+rapids-logger "Installing wheel"
+
+pip install pynvjitlink-0.1.0-cp310-cp310-manylinux_2_35_x86_64.whl
+
+rapids-logger "Building tests"
+
+cd test_binary_generation && make
+
+rapids-logger "Running Tests"
+
+cd ..
+conda install -y pytest
+py.test pynvjitlink/tests
```

Comment:
I think I would prefer to run tests in a separate job that depends on the build job.

Reply:
Roger that, I was wondering the same thing. Do we think it's worth doing both in this one PR or would it be better to break off the testing phase into its own PR to avoid muddying things?

Reply:
Let's do it in this PR. We could use a wider test matrix than our build matrix.